cups.pc.in

```diff
@@ -0,0 +1,40 @@
+#cups pkg-config file
+
+
+VERSION="@CUPS_VERSION@"
+APIVERSION="2.3"
+BUILD="@CUPS_BUILD@"
+
+prefix=@prefix@
+exec_prefix=@exec_prefix@
+bindir=@bindir@
+includedir=@includedir@
+libdir=@libdir@
+datarootdir=@datadir@
+#datadir=@datadir@
+sysconfdir=@sysconfdir@
+cups_datadir=@CUPS_DATADIR@
+cups_serverbin=@CUPS_SERVERBIN@
+cups_serverroot=@CUPS_SERVERROOT@
+INSTALLSTATIC=@INSTALLSTATIC@
+
+
+# flags for compiler and linker...
+CFLAGS=" -I${includedir}"
+LDFLAGS="@EXPORT_LDFLAGS@ -L${libdir}"
+LIBS="@LIBGSSAPI@ @DNSSDLIBS@ @EXPORT_SSLLIBS@ @LIBZ@ @LIBS@"
+
+Name: "CUPS"
+Description: "CUPS pkg-config file"
```

Comment:
The description should probably say something like "CUPS API library"

We should also use Requires: for dependent libraries rather than embed them.